c++/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
This probably shouldn't be 2.8, since it didn't work with 2.8? Or did it? To be safe put it to whatever version you have now.